package.json

```diff
@@ -8,7 +8,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "node_modules/.bin/mocha --reporter spec --recursive"
+    "test": "mocha -R spec --recursive",
```

Comment:
How come you removed the reference to `node_modules/.bin/mocha`? Now dependencies have to be installed globally in order to work correctly